examples/preproc3-NVT_E2E.ipynb

```diff
@@ -0,0 +1,890 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"CUDA_DEVICE_ORDER\"] = \"PCI_BUS_ID\"\n",
+    "os.environ[\"CUDA_VISIBLE_DEVICES\"] = \"3\""
```

Comment:
I'd like to leave these out of notebooks, and just rely on them being set at the OS level (like 'CUDA_VISIBLE_DEVICES=3' won't work for many users that don't have a 4 gpu machine).

Reply:
Done.